src/mono/mono/metadata/assembly.c

```diff
@@ -2715,8 +2754,8 @@ mono_assembly_request_open (const char *filename, const MonoAssemblyOpenRequest
 
 	// If VM built with mkbundle
 	loaded_from_bundle = FALSE;
-	if (bundles != NULL) {
-		image = mono_assembly_open_from_bundle (load_req.alc, fname, status, refonly);
+	if (bundles != NULL || satellite_bundles != NULL) {
+		image = mono_assembly_open_from_bundle (load_req.alc, fname, status, refonly, NULL);
```

Comment:
Another case where we aren't culture-aware. I think this is fine, but it should have a comment noting that.